src/components/Task/Difficulty/DifficultyLabel/DifficultyLabel.scss

```diff
@@ -1,14 +1,23 @@
 @import './src/css/styles';
 
-.difficulty-label {
+.difficulty-label-container {
+  position: relative;
+  width: 5rem;
+}
+
+.difficulty-label-item {
   align-items: center;
   border-radius: 1rem;
   border-style: solid;
   border-width: 1px;
   color: $color-dark;
-  display: flex;
+  display: inline-flex;
   height: 1.5rem;
   justify-content: center;
-  padding: 0 1rem;
+  left: 50%;
+  padding: 0 0.5rem;
+  position: absolute;
   text-align: center;
+  top: 50%;
```

Comment:
bad way to center things. now you can have the problem, that the fontsize changes and the font is now bigger than the box around

Reply:
if you have a better solution, I would be happy. I would like a fixed size of the component and then I have to align it manually because `display: flex` does not work correctly